main.nf

```diff
@@ -1,114 +1,26 @@
 #!/usr/bin/env nextflow
```

Comment:
Same comments here as for hello world. Can you keep the comment lines in your scripts so we know what the code sections are doing at a basic level.